02 - Memtables, Joins, and Selectors.ipynb

```diff
@@ -463,7 +463,7 @@
    "outputs": [],
```

Comment:
<p>This example is a bit confusing since this condition is always FALSE, maybe adding a comment saying that this performs a FULL JOIN  that returns all rows in both tables, using NULLs for values with no match.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis-tutorial/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>